composer.json

```diff
@@ -2,6 +2,7 @@
 	"name": "apache-solr-for-typo3/solrgrouping",
 	"type": "typo3-cms-extension",
 	"description": "Apache Solr for TYPO3 - Result Grouping",
+	"version": "1.1.0",
```

Comment:
Please do not add a version property in composer.json, it just causes trouble. Packagist will use gift tags, that's all we need